frontend/src/features/SideWindow/MissionForm/hooks/useListenMissionEventUpdatesById.ts

```diff
@@ -0,0 +1,23 @@
+import { useContext, useMemo } from 'react'
+
+import { MissionEventContext } from '../../../../context/MissionEventContext'
+import { usePreviousNotNull } from '../../../../hooks/usePreviousNotNull'
+
+import type { Mission } from '../../../../domain/entities/mission/types'
+
+export function useListenMissionEventUpdatesById(missionId: number | string | undefined) {
```

Comment:
`useListenToMissionEventUpdatesById()` ?

Reply:
+1 pour le "[...]To[...]"